ui/raidboss/data/06-ew/raid/p5s.ts

```diff
@@ -115,6 +141,72 @@ const triggerSet: TriggerSet<Data> = {
       durationSeconds: 5,
       response: Responses.getBackThenFront(),
     },
+    {
+      id: 'P5S Topaz Ray Collect',
```

Comment:
What does using a collector here gain over just putting this in the below trigger's `run` instead?

Reply:
I guess there is none as long as they return if the array's length isn't expected. I added the safeguards after writing the skeleton. However, I think the next iteration requires getCombatants, so it will still need a collector.